libsplinter/Cargo.toml

```diff
@@ -194,6 +195,7 @@ store-factory = []
 trust-authorization = []
 ws-transport = ["tungstenite"]
 zmq-transport = ["zmq"]
+node-id-store = []
```

Comment:
Missed this in the alphabetical ordering

Reply:
fixed, thanks for catching it